apps/nextjs-website/src/lib/webinars/webinarQuestions.ts

```diff
@@ -0,0 +1,92 @@
+import { pipe } from 'fp-ts/lib/function';
+import * as RTE from 'fp-ts/lib/ReaderTaskEither';
+import * as RA from 'fp-ts/lib/ReadonlyArray';
+import * as TE from 'fp-ts/lib/TaskEither';
+import * as R from 'fp-ts/lib/Reader';
+import * as E from 'fp-ts/lib/Either';
+import {
+  DynamoDBClient,
+  PutItemCommand,
+  QueryCommand,
+} from '@aws-sdk/client-dynamodb';
+import {
+  WebinarQuestionDynamodbCodec,
+  makeDynamodbItemFromWebinarQuestion,
+  makeWebinarQuestionFromDynamodbItem,
+  makeWebinarQuestionListQueryCondition,
+} from './dynamodb/codec';
+
+export type WebinarEnv = {
+  readonly questionLifetimeInSeconds: number;
+  readonly dynamoDBClient: DynamoDBClient;
+  readonly nowDate: () => Date;
+};
+
+export type InsertWebinarQuestion = Omit<
+  WebinarQuestion,
+  'createdAt' | 'expireAt'
+>;
+
+export type WebinarQuestion = {
+  readonly webinarId: string;
+  readonly givenName: string;
+  readonly familyName: string;
+  readonly question: string;
+  readonly createdAt: Date;
+  readonly expireAt: Date;
+};
+
+export const insertWebinarQuestion = (question: InsertWebinarQuestion) =>
```

Comment:
At the moment the type is `RTE.ReaderTaskEither<Pick<WebinarEnv, "dynamoDBClient" | "nowDate" | "questionLifetimeInSeconds">, Error, undefined>`, so the output is `Either<Error, undefined>`. It might make sense to return the inserted question, though I'm uncertain whether it will be used